docs/sources/get-started/quick-start.md

```diff
@@ -2,160 +2,248 @@
 title: Quickstart to run Loki locally
 menuTitle: Loki quickstart
 weight: 200
-description: How to create and use a simple local Loki cluster for testing and evaluation purposes.
+description: How to create and use a local Loki cluster for testing and evaluation purposes.
+killercoda:
```

Comment:
Love how clean and easy this is and you can see where it all plugs into the JSON config on the repo side. Do you think it would still be possible to add
```json
  "backend": {
    "imageid": "ubuntu"
  }
```
something like
```yml
backend: ubuntu
```

Reply:
I'm not sure I understand, sorry. I've mapped things directly from the Killercoda `index.json`.

Are you thinking of simplifying the 

```
  backend:
    imageid: ubuntu
```

in https://github.com/grafana/loki/pull/13234/files#diff-427246e156e0ad4ae83298b71cb7f25000ab0c06fbf2817c211eb00b1e5e67d2R12-R13

with

```
  backend: ubuntu
```